profiles/Sink/Sink_HdmiCec.yaml

```diff
@@ -0,0 +1,9 @@
+Device:
+  Type: sink
+  Name: element
+
+# cec Device Type 0- None, 1 - Sink, 2 - Source
+cecDeviceType: 2
```

Comment:
format is not valid.

hdmicec:
 type: sink
 name: element
 cecDeviceType: 2

Reply:
updated profile file and ensured the yaml is valid using validator.